app/src/components/inputs/inputCheckbox/checkIcon/checkIcon.scss

```diff
@@ -27,6 +27,10 @@
   border-color: $COLOR--gray-60;
   box-sizing: border-box;
 
+  &.ghost {
+    border-color: $COLOR--topaz;
```

Comment:
Is it OK that square will have the same border in all states: plain, hover, clicked? Have you discussed this with the designer?